site/src/utils/hasura/mutation.js

```diff
@@ -0,0 +1,72 @@
+const gql = require("graphql-tag")
+const { ApolloClient } = require("apollo-boost")
+const { fetch } = require("isomorphic-fetch")
+const { HttpLink } = require("apollo-boost")
+const { InMemoryCache } = require("apollo-boost")
+const { transformOrganization } = require("./transformHelpers")
+
+require("dotenv").config({
+  path: `../../../../.env.${process.env.NODE_ENV}`,
+})
+
+// Gatsby does not support mutations because it is designed to expect
+// mutations to occur only on the client and not at build time.
+// See: https://github.com/gatsbyjs/gatsby/issues/13704
+// In order to run mutations at build time, we are using Apollo.
+
+const apolloClient = new ApolloClient({
+  link: new HttpLink({
+    uri: process.env.GRAPHQL_URI,
+    fetch,
+  }),
+  cache: new InMemoryCache(),
+})
+
+const mutation = gql`
+  mutation AddOrganizations($objects: [organizations_insert_input!]!) {
```

Comment:
Nitpick, could we call this `mutation MirrorOrganizations` since it won't always add?